lib/manageiq/appliance_console/cli.rb

```diff
@@ -116,6 +126,11 @@ def parse(args)
         opt :force_key, "Forcefully create encryption key", :type => :boolean, :short => "f"
         opt :sshlogin,  "SSH login",         :type => :string,                 :default => "root"
         opt :sshpassword, "SSH password",    :type => :string
+        opt :replication, "Configure database replication as primary or standby", :type => :string, :short => :none
+        opt :primary_host, "Primary database host IP address", :type => :string, :short => :none
+        opt :standby_host, "Standby database host IP address", :type => :string, :short => :none
+        opt :repmgrd, "Configure Replication Manager (repmgrd) for automatic failover", :type => :bool, :short => :none
```

Comment:
I think we can make this option name better, if people don't know anything about `repmgrd`, it won't be clear what this does.

I think `auto_failover` would be better.